.github/workflows/build.yml

```diff
@@ -107,3 +107,41 @@ jobs:
             build/reports
             catalogs/**/*.log
             catalogs/**/*.tar
+
+  spark-connector-build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        scala-version: [ 2.12, 2.13 ]
+    timeout-minutes: 30
+    needs: changes
+    if: needs.changes.outputs.source_changes == 'true'
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: actions/setup-java@v4
+        with:
+          java-version: 8
+          distribution: 'temurin'
+          cache: 'gradle'
+
+      - name: Free up disk space
+        run: |
+          dev/ci/util_free_space.sh
+
+      - name: Build with Gradle
```

Comment:
How about adding a separate step in `build` named like `Build spark connector with scala 2.13` to build and test scala 2.13 packages.

Reply:
@FANNG1 just test scala2.13, not considering scala2.12?

Reply:
updated, only support scala2.13